governance/GOVERNANCE.md

```diff
@@ -0,0 +1,116 @@
+# ORAS Governance
+
+The following document outlines ORAS project governance.
+
+## The ORAS Project
+
+The ORAS project consists of several repositories known as sub-projects that enable community cohorts to experiment and implement solutions across the scope of the project.
+
+## Owners Structure
+
+There are two types of owners in the ORAS project organized hierarchically. ORAS [org owners][oras-project-owners] oversee the overall project and its health. Subproject owners focus on a single repository, a group of related repositories, a service (e.g., a website), or subproject to support the other subprojects (e.g., marketing or community management). 
+
+Changes in Oras Org owners have to be announced via an [oras-project/community/issues][oras-issues-new]. Changes to sub-project owners are to be announced via the appropriate sub-project issue.
+
+### Owner Responsibility
+
+ORAS owners adhere to the requirements and responsibilities set forth in the respective [ORAS Org Owners](#oras-org-owners) and [Subproject Owners](#subproject-owners). They further pledge the following:
+
+- To act in the best interest of the project and subprojects
+- To ensure that project and subproject development and direction is a function of community needs
+- To never take any action while hesitant that it is the right action to take
+- To fulfill the responsibilities outlined in this document and its dependents
+
+### ORAS Org Owners
+
+The [ORAS Org owners][oras-project-owners] are responsible for:
+
+- Maintaining the mission, vision, values, and scope of the project
+- Refining the governance and charter as needed
+- Making project level decisions
+- Resolving escalated project decisions when the subproject owners responsible are blocked
+- Managing the ORAS brand
+- Controlling access to ORAS assets such as source repositories, hosting, project calendars
+- Deciding what subprojects are part of the ORAS project
+- Deciding on the creation of new subprojects
+- Overseeing the resolution and disclosure of security issues
+- Managing financial decisions related to the project
+
+Changes to org owners use the following:
+
+- Any subproject owner is eligible for a position as an org owner
+- No one company or organization can employ a simple majority of the org owners
```

Comment:
Can we change this to "should employ"? I don't want us to be in a situation where one person leaves right after someone else from one company joins, and we suddenly are in violation, just because one company's employees stepped up for a bit. The rules should nudge us in the right direction, not suddenly find us in violation despite (or because of) extra good will.

Reply:
@lachie83 and I talked about this a bit. The concern isn't that a company would be in violation if folks leave and gaps need to be filled. It's more that this creates an action item to resolve. If it's worded as _should employ_ it leaves it open to interpretation where a company can say SHOULD isn't MUST, so they would have leeway.

Reply:
Fair enough. It is weighing the balance of "creating action items" vs "being in violation".